### ISSUE TITLE
Protein Interactions Feature Requests

### ISSUE THREAD

**Issue by bmpbowen** [ClinGen/gene-and-variant-curation-tools#366]
We've received a request to update the field related to "Has this gene or genes been implicated in the above disease?" in the Protein Interactions section of the GCI. Example: https://curation-test.clinicalgenome.org/curation-central/7dbe9416-aa81-40c1-9aec-b2cfad3a501a/annotation/d61cbe3b-3cc3-4be3-9092-3f15682c5b1e/experimental-curation/c906268a-7646-4d47-9a73-d4f3ff36c0ff/edit/

A user observed that evidence related to this field is added to the overall [score] (https://curation-test.clinicalgenome.org/curation-central/7dbe9416-aa81-40c1-9aec-b2cfad3a501a/gene-disease-evidence-summary/?preview=yes) even when the box was is not checked. Going forward, users would like this field to directly relate to scoring in the following ways:

1. Make “Has this gene or genes been implicated in the above disease?" a required field.
2. Change from one checkbox to two radio buttons: 1 Yes, 2 No
3. Edit Warning Box text:
Old text: “Warning: not checking the above box indicates this criteria has not been met for this evidence; this should be taken into account during its evaluation.”
New text: “Warning: Selecting “No” for the above required question indicates this criteria has not been met for this evidence; therefore this protein interaction will not be scored.”

![image](https://github.com/ClinGen/gene-and-variant-curation-tools/assets/139911863/bfefecdc-7684-43f3-bd35-ceacd6ec1d6a)

4. Disallow "Score" as a dropdown Status option when the "Has this gene or genes been implicated in the above disease?" box is unanswered; show an error instead of Saving: “Missing field required”.
5. Disallow “Score” as a dropdown Status option when the "Has this gene or genes been implicated in the above disease?" box is marked No; show an error instead of Saving: "Only genes implicated in the above disease are eligible to be scored for protein interactions"
If 4 and 5 can't be implemented, an alternative would be to set default score to 0 when the question "Has this gene or genes been implicated in the above disease?" is answered "No"

In the future, we could try to automate the answer to this question by cross-referencing other systems, but that would require consistent disease names across data sources or a way to check parent/sibling disease terms (GCI, GenCC) which does not yet exist, so tabling that suggestion for now.

**Comment by wrightmw** [comment]
@bmpbowen This ticket requires further internal discussion with our dev team. There are impacts on the logic within the UI which may mean this needs to be re-specified.

**Comment by bmpbowen** [comment]
@wrightmw certainly! Iplanned to discuss at our next Stanford Clingen meeting. Will add to that agenda.

**Comment by bmpbowen** [comment]
Proposed alternative to the above fix is: Instead of yes/no, have the check box auto-select once a gene is entered in the first field. This way no additional changes are required.
